src/components/IndividualPageContent/ContentRow.tsx

```diff
@@ -9,22 +9,27 @@ type ContentRowProps = {
   value: React.ReactNode;
   tooltip?: React.ReactNode;
   i?: any;
+  container?: boolean;
+  sx?: SxProps<Theme>;
 };
 
 export default function ContentRow({
```

Comment:
do you mind holding off the style change in this component? I think it's a design discussion that do we want to introduce different content row style for the validator page. If there's any change, let's do it after the design gets finalized.

Reply:
yes in the new revision, will remove these design changes!